jumpstarter/states.py

```diff
@@ -30,6 +30,17 @@ class ActorState(Enum):
     crashed = auto()
 
 
+class ActorStateTriggers(str, Enum):
```

Comment:
So we need this just to avoid magic strings?

Reply:
Yeah. It is not really necessary. But it does help keep code slightly less verbose since the enum instance itself can act as a string.

Reply:
Happy to take it out if you want.

Reply:
There has been a debate back and forth about whether to have this feature or not. Looks like they finally just added it. 

https://github.com/python/cpython/commit/0063ff4e583505e69473caa978e476ea4c559b83